EncoreTickets.SDK/Utilities/Enums/AuthenticationMethod.cs

```diff
@@ -13,6 +13,11 @@ public enum AuthenticationMethod
         /// <summary>
         /// Authentication based on username and password.
         /// </summary>
-        Basic
+        Basic,
+
+        /// <summary>
+        /// Authentication based on API key.
+        /// </summary>
+        ApiKey
```

Comment:
I'm not sure if this is the right way to call this entry. The API key in general can be sent in multiple ways but it seems we are currently only using JWTs without expiration and call them "API keys"